test/flows.spec.js

```diff
@@ -326,13 +326,15 @@ describe('Flows', function() {
         .slice(2);
 
       function onRequest(request) {
-        expect(request).to.eql({
-          api_version: 'latest',
-          idempotency_key: idempotencyKey,
-          method: 'POST',
-          path: '/v1/charges',
-          account: connectedAccountId,
-        });
+        expect(request.api_version).to.equal('latest');
+        expect(request.idempotency_key).to.equal(idempotencyKey);
+        expect(request.account).to.equal(connectedAccountId);
+        expect(request.method).to.equal('POST');
+        expect(request.path).to.equal('/v1/charges');
+        expect(request.request_start_time).to.be.within(
+          Date.now() - 30000,
+          Date.now()
```

Comment:
Just a note that in general, I don't like tests that rely on a real clock and prefer using dependency injection or even mocks/stubs to make the test more deterministic... Not sure if we have an established pattern for doing this in stripe-node. We can probably keep this as-is and revisit if the test ever becomes flaky.

Reply:
I concur with you on your sentiments on tests like this. I went with this approach because I saw this pattern used for testing the elapsed time so I figured it was what Stripe expects/prefers. Perhaps just a type check of these values to ensure they are a number would be a better option?

Reply:
While I generally also dislike relying on the real clock, I don't think we have delightful mocking available at this time.

That said, we can probably set the starting bound to be a `Date.now()` set at the top of the test, which should be both more accurate and robust. Similarly, below, in an ideal world we'd check that `response.request_end_time` is greater than `request.request_start_time` (which would require adding an `onRequest` handler).

Reply:
Good suggestions @rattrayalex-stripe, I went ahead and added the request_start_time to the response event as well as that lets me more easily implement your request_end_time greater that request_start_time test as well as improve the elapsed test to not be vulnerable to a bizarre long request. Plus I suspect for some people's usage it will be beneficial for adding logging of the start and end time of any Stripe request. I also made the set lower bound at start of test change. This should be ready for you to review again. Thanks!

Reply:
Great thinking! Thanks!